liblangutil/Exceptions.h

```diff
@@ -23,27 +23,39 @@
 
 #pragma once
 
-#include <libsolutil/Exceptions.h>
+#include <liblangutil/SourceLocation.h>
 #include <libsolutil/Assertions.h>
 #include <libsolutil/CommonData.h>
-#include <liblangutil/SourceLocation.h>
+#include <libsolutil/Exceptions.h>
 
+#include <memory>
 #include <string>
 #include <utility>
 #include <vector>
-#include <memory>
 
 namespace solidity::langutil
 {
 class Error;
 using ErrorList = std::vector<std::shared_ptr<Error const>>;
 
-struct CompilerError: virtual util::Exception {};
-struct StackTooDeepError: virtual CompilerError {};
-struct InternalCompilerError: virtual util::Exception {};
-struct FatalError: virtual util::Exception {};
-struct UnimplementedFeatureError: virtual util::Exception {};
-struct InvalidAstError: virtual util::Exception {};
+struct CompilerError: virtual util::Exception
+{
+};
+struct StackTooDeepError: virtual CompilerError
+{
+};
+struct InternalCompilerError: virtual util::Exception
+{
+};
+struct FatalError: virtual util::Exception
+{
+};
+struct UnimplementedFeatureError: virtual util::Exception
+{
+};
+struct InvalidAstError: virtual util::Exception
+{
+};
```

Comment:
Can you avoid this change? I assume this was done by the IDE, but this is unnecessary for the PR.

Reply:
Fixed 👍